geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/sortedset/AbstractZUnionStoreIntegrationTest.java

```diff
@@ -70,6 +70,44 @@ public void shouldError_givenWrongKeyType() {
                 .hasMessage("WRONGTYPE " + RedisConstants.ERROR_WRONG_TYPE);
   }
 
+  @Test
+  public void shouldError_givenTwoSortedSetKeysAndWrongKeyType() {
```

Comment:
This test name might be a little clearer as something like "shouldReturnWrongTypeError_givenNonSortedSetKeyAsThirdSourceKey"